src/rpc/RPCHelpers.cpp

```diff
@@ -458,19 +459,20 @@ toExpandedJson(Backend::TransactionAndMetadata const& blobs)
     auto [txn, meta] = deserializeTxPlusMeta(blobs, blobs.ledgerSequence);
     auto txnJson = toJson(*txn);
     auto metaJson = toJson(*meta);
-    insertDeliveredAmount(metaJson, txn, meta);
+    insertDeliveredAmount(metaJson, txn, meta, blobs.date);
     return {txnJson, metaJson};
 }
 
 bool
 insertDeliveredAmount(
     boost::json::object& metaJson,
     std::shared_ptr<ripple::STTx const> const& txn,
-    std::shared_ptr<ripple::TxMeta const> const& meta)
+    std::shared_ptr<ripple::TxMeta const> const& meta,
+    uint32_t date)
 {
     if (canHaveDeliveredAmount(txn, meta))
```

Comment:
Do we also need to change `canHaveDeliveredAmount`?

Reply:
Don't think so, that function is just looking at the transaction to see if its the type of transaction that could have a delivered amount.